packages/kit/src/runtime/server/endpoint.js

```diff
@@ -37,24 +38,27 @@ export default async function render_route(request, route) {
 			headers = lowercase_keys(headers);
 			const type = headers['content-type'];
 
+			const is_type_binary = isContentTypeBinary(type);
+			/** @type {import('types/hooks').StrictBody} */
+			let normalized_body;
```

Comment:
I would move this line back down where it was originally. No reason to declare it higher up since it's not used yet